src/app/services/focus.service.ts

```diff
@@ -17,8 +17,18 @@ export class FocusService {
 
     restoreFocus(): void {
       if (this.lastFocusedElement) {
-        this.lastFocusedElement.focus();
-        this.lastFocusedElement = null;
+        setTimeout(() => {
+          const dataTestValue = this.lastFocusedElement.getAttribute('data-test');
```

Comment:
Is it possible when the last focused element does not have a `data-test` attribute?

Reply:
> Is it possible when the last focused element does not have a `data-test` attribute?

we have a linter for such elements, they should have data-test.